comment-sample/package-lock.json

```diff
@@ -1,8 +1,1751 @@
 {
 	"name": "vscode-comment-api-example",
 	"version": "0.0.1",
```

Comment:
Generally speaking, we prefer to update the lock files ourselves. Could you please revert the lock file changes? Then, whoever merges the PR will ensure the lock files are updated before merging.

Reply:
Sounds good. I would recommend including this somewhere since the only thing I saw regarding this was in [Sample Guideline](https://github.com/microsoft/vscode-extension-samples/blob/main/.github/SAMPLE_GUIDELINE.md#4-dependencies) where it just recommended using `package-lock.json` instead of `yarn.lock`.